pkg/backup-session/backup_session.go

```diff
@@ -0,0 +1,70 @@
+package backup_session
+
+import (
+	"github.com/appscode/go/crypto/rand"
+	api_v1beta1 "github.com/appscode/stash/apis/stash/v1beta1"
+
+	//cs "github.com/appscode/stash/client/clientset/versioned"
+	cs "github.com/appscode/stash/client/clientset/versioned/typed/stash/v1beta1"
+	"github.com/appscode/stash/client/clientset/versioned/typed/stash/v1beta1/util"
+	"github.com/tamalsaha/go-oneliners"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/apis/core"
+)
+
+type Controller struct {
+	Options
+	k8sClient          kubernetes.Interface
+	stashv1beta1Client cs.StashV1beta1Interface
+}
+
+type Options struct {
+	Name      string
+	Namespace string
+}
+
+func New(k8sClient kubernetes.Interface, stashv1betaClient cs.StashV1beta1Interface, opt Options) *Controller {
+	return &Controller{
+		k8sClient:          k8sClient,
+		stashv1beta1Client: stashv1betaClient,
+		Options:            opt,
+	}
+}
+
+func (c *Controller) CreateBackupSessionCrd() error {
```

Comment:
Rename `CreateBackupSessionCrd()` to `CreateBackupSession()`

Reply:
resolved